src/components/installer/installer.ts

```diff
@@ -0,0 +1,215 @@
+import { Shell, Platform } from '../../shell';
+import { Errorable } from '../../wizard';
+import { exec } from 'child_process';
+
+export async function installKubectl(shell: Shell) : Promise<Errorable<void>> {
+    const tool = 'kubectl';
+    const installFolder = getInstallFolder(shell, tool);
+    await mkdir(installFolder);
+    switch (shell.platform()) {
+        case Platform.Windows:
+        {
+            // Instruction say always to use this link for Windows, not to check the version first
+            const kubectlUrl = 'https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/windows/amd64/kubectl.exe';
+            const downloadFile = path.join(installFolder, 'kubectl.exe');
+            const downloadResult = await downloadTo(kubectlUrl, downloadFile);
+            if (!downloadResult.succeeded) {
+                return { succeeded: false, result: null, error: [`Failed to download kubectl: ${downloadResult.error[0]}`] };
+            }
+            addPathToConfig(`vs-kubernetes.${tool}-path`, downloadFile);
+            return { succeeded: true, result: null, error: [] };
+        }
+        case Platform.Linux:
+        case Platform.MacOS:
+        {
+            const version = await getStableKubectlVersion();
+            if (!version.succeeded) {
+                return { succeeded: false, result: null, error: version.error };
+            }
+            const os = platformUrlString(shell.platform());
+            const kubectlUrl = `https://storage.googleapis.com/kubernetes-release/release/${version.result.trim()}/bin/${os}/amd64/kubectl`;
+            const downloadFile = path.join(installFolder, 'kubectl');
+            const downloadResult = await downloadTo(kubectlUrl, downloadFile);
+            if (!downloadResult.succeeded) {
+                return { succeeded: false, result: null, error: [`Failed to download kubectl: ${downloadResult.error[0]}`] };
+            }
+            fs.chmodSync(downloadFile, '0777');
+            addPathToConfig(`vs-kubernetes.${tool}-path`, downloadFile);
+            return { succeeded: true, result: null, error: [] };
+        }
+    }
+}
+
+async function getStableKubectlVersion() : Promise<Errorable<string>> {
+    const downloadResult = await download('https://storage.googleapis.com/kubernetes-release/release/stable.txt');
+    if (!downloadResult.succeeded) {
+        return { succeeded: false, result: '', error: [`Failed to establish kubectl stable version: ${downloadResult.error[0]}`] };
+    }
+    const version = fs.readFileSync(downloadResult.result, 'utf-8');
+    fs.unlinkSync(downloadResult.result);
+    return { succeeded: true, result: version, error: [] };
+}
+
+export async function installHelm(shell: Shell) : Promise<Errorable<void>> {
+    const tool = 'helm';
+    const executable = formatBin(tool, shell.platform());
+    const urlTemplate = 'https://storage.googleapis.com/kubernetes-helm/helm-v2.8.2-{os_placeholder}-amd64.tar.gz';
```

Comment:
At some point in the future, we might be able to use the GitHub API to find the latest release tag for the release. Definitely not necessary for this PR, though